index.html

```diff
@@ -214,22 +214,22 @@ <h4 class="connect">let's connect</h4>
         </ul>
     </section>
 
-    <section id="contact-form">
+    <footer id="contact-form">
         <div class="contact-container">
             <h2>contact me</h2>
             <p>
                 If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
                 I’d love to help with it!
             </p>
-            <form action="#">
+            <form  action="https://formspree.io/f/mpzebpge" method="POST">
                 <div class="form-inputs">
-                    <input class="form-items" type="text" placeholder="Lorem Ipsum">
-                    <input class="form-items" type="email" name="" id="" placeholder="loremipsum@gmail.com">
-                    <textarea name="" id="" cols="30" rows="10" placeholder="Write your message here..."></textarea>
+                    <input maxlength="30"  class="form-items" type="text" name="name" placeholder="Lorem Ipsum" required>
+                    <input class="form-items" type="email" name="email" placeholder="loremipsum@gmail.com" required>
+                    <textarea maxlength="500" name="message" cols="30" rows="10" placeholder="Write your message here..." required></textarea>
                 </div>
-                <button class="btn cont">get in touch</button>
+                <button type="submit" class="btn cont">get in touch</button>
             </form>
```

Comment:
- [ ] Your form is working fine well done :clap: but your design is not same as Figma design so kindly stick to the Figma design as much as possible :+1: check below :point_down: the comparison between your designs and Figma design :slightly_smiling_face: 

| Your Design | Figma Design |
|--------|--------|
| ![Screenshot from 2023-05-02 14-58-55](https://user-images.githubusercontent.com/45226848/235638659-bdaa210e-e089-4c9a-8204-15eccc52ec6a.png) |  ![Screenshot from 2023-05-02 14-57-40](https://user-images.githubusercontent.com/45226848/235638814-4eb4f867-959f-4ca4-aef0-587c7be5cb4e.png)  |
| ![Screenshot from 2023-05-02 14-59-07](https://user-images.githubusercontent.com/45226848/235638716-f7cd9cd5-a393-4d09-8038-dc8792b0d934.png) | ![Screenshot from 2023-05-02 14-57-56](https://user-images.githubusercontent.com/45226848/235638837-8c072f6a-10c0-4f75-acb0-47e88028adbf.png) | 

- [ ] If you check your design the background colour is totally different from the Figma design so kindly fix it :slightly_smiling_face: 
- [ ] Also, if you check your button the colour of your button is also different from the Figma design so kindly use same color as mentioned in Figma design :+1: